apt.txt

```diff
@@ -43,3 +43,18 @@ libblas-dev
 liblapack-dev
 libgraphviz-dev
 libsoxr-dev
+libgl1-mesa-dev
+libvulkan-dev
+libx11-xcb-dev
+libxcb-dri2-0-dev
+libxcb-glx0-dev
+libxcb-icccm4-dev
+libxcb-keysyms1-dev
+libxcb-randr0-dev
+libxrandr-dev
+libxxf86vm-dev
+mesa-common-dev
+libglfw3-dev
+libglew-dev
+libglm-dev
+libfreetype6-dev
```

Comment:
@vvasco @AlexAntn @vtikha we have to check if this change does not break our docker images

But is ok to commit it, we will fix in case on our side!

Reply:
Just to understand, what could be a failure mode of just installing new apt packages from the official repos?

Reply:
We have some images that build from source code the mesa libraries, but I think that for those we do not use the `apt.txt ` for installing the packets.
But if it is so we can just edit those images and don't use apt.txt